### ISSUE TITLE
Unhandled exception when filtering Enum column

### ISSUE THREAD

**Issue by bryanrcarlson** [IvanJosipovic/BlazorTable#157]
Using version 1.12.0  (downgrading to version 1.11.1 fixed this issue)

When I filter a column with an Enum value (Caf.Midden.Utils.Models.Zones) I get the following error:

```
Microsoft.AspNetCore.Components.WebAssembly.Rendering.WebAssemblyRenderer[100]
      Unhandled exception rendering component: The binary operator NotEqual is not defined for the types 'Caf.Midden.Utils.Models.Zones' and 'System.Object'.
System.InvalidOperationException: The binary operator NotEqual is not defined for the types 'Caf.Midden.Utils.Models.Zones' and 'System.Object'.
  at (wrapper managed-to-native) System.Reflection.RuntimeMethodInfo.InternalInvoke(System.Reflection.RuntimeMethodInfo,object,object[],System.Exception&)
  at System.Reflection.RuntimeMethodInfo.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) <0x262b9e8 + 0x000ce> in <filename unknown>:0 
--- End of stack trace from previous location where exception was thrown ---
```

Relevant Razor code:

```
 <Column TableItem="Dataset" Title="Zone" Field="@(d => d.Zone)" Sortable="true" Filterable="true" Width="5%" />
```

Great component, btw. Thanks!

**Comment by IvanJosipovic** [comment]
Hey,

Can you please create a page that fully reproduces this issue?
See [Example](https://github.com/IvanJosipovic/BlazorTable/blob/master/src/BlazorTable.Sample.Shared/Bugs/152.razor)

**Comment by IvanJosipovic** [comment]
Thanks for the repro, this issue is fixed in 1.12.1
